### ISSUE TITLE
Can't enqueue staticmethods

### ISSUE THREAD

**Issue by pwws** [rq/rq#1457]
Worker's can't run class methods decorated with `staticmethod`. Consider the following code and let's assume it's in `staticmethod_test` module:
```python
class TestClass:
    @staticmethod
    def test_job():
        return 'Test static method'
```
Enqueuing `TestClass.test_job` is successful:
```python
queue.enqueue_call(staticmethod_test.TestClass.test_job)
# Job('e8fe13af-b996-401e-a4a9-431e72c5bfa4', enqueued_at=datetime.datetime(2021, 5, 4, 10, 55, 45, 44138))
```
but a `worker` can't handle it:
```python
12:55:45 default: staticmethod_test.test_job() (e8fe13af-b996-401e-a4a9-431e72c5bfa4)
12:55:45 Traceback (most recent call last):
  File "/home/xxx/prog/rq/rq/worker.py", line 1021, in perform_job
    rv = job.perform()
  File "/home/xxx/prog/rq/rq/job.py", line 707, in perform
    self._result = self._execute()
  File "/home/xxx/prog/rq/rq/job.py", line 730, in _execute
    result = self.func(*self.args, **self.kwargs)
  File "/home/xxx/prog/rq/rq/job.py", line 219, in func
    return import_attribute(self.func_name)
  File "/home/xxx/prog/rq/rq/utils.py", line 130, in import_attribute
    return getattr(module, attribute)
AttributeError: module 'staticmethod_test' has no attribute 'test_job'
```

Apparently, I'm not the only one that had this problem: https://github.com/rq/rq/issues/582#issuecomment-368455932, https://github.com/rq/django-rq/issues/248

**Comment by pwws** [comment]
I have prepared a fix for this problem: https://github.com/rq/rq/pull/1458

**Comment by selwin** [comment]
Fixed in #1458
